src/components/SearchMain/index.js

```diff
@@ -27,32 +33,65 @@ export default class SearchMain extends PureComponent {
 
   render() {
     const { model } = this.props;
+    const {
```

Comment:
For consistency sake, define canCreate here

Reply:
All done.